src/main/java/life/qbic/model/download/QbicDataDownloader.java

```diff
@@ -305,38 +302,37 @@ private void downloadFile(DataSetFile dataSetFile, Path prefix) throws IOExcepti
     options.setRecursive(false);
     IDataSetFileId fileId = dataSetFile.getPermId();
     InputStream stream =
-        this.dataStoreServer.downloadFiles(
-            sessionToken, Collections.singletonList(fileId), options);
+            this.dataStoreServer.downloadFiles(
+                    sessionToken, Collections.singletonList(fileId), options);
     DataSetFileDownloadReader reader = new DataSetFileDownloadReader(stream);
     DataSetFileDownload file;
 
     while ((file = reader.read()) != null) {
       InputStream initialStream = file.getInputStream();
       CheckedInputStream checkedInputStream = new CheckedInputStream(initialStream, new CRC32());
       if (file.getDataSetFile().getFileLength() > 0) {
-        final Path filePath = OutputPathFinder.determineFinalPathFromDataset(file.getDataSetFile(),conservePaths);
-
-        File newFile = OutputPathFinder.determineOutputDirectoryForData(outputPath, prefix, filePath);
-        OutputStream os = new FileOutputStream(newFile);
-
+        final Path filePath = OutputPathFinder.determineFinalPathFromDataset(file.getDataSetFile(), conservePaths);
+        final Path newPath = OutputPathFinder.determineOutputDirectory(outputPath, prefix, filePath);
```

Comment:
In Java, it is always good to avoid using new as it is a keyword. So I would suggest renaming the created path here to `outputDirectory` or something similar.

NIT (This is nit-picky so you do not have to do this to get the PR approved)